build.sh

```diff
@@ -174,6 +174,14 @@ build_watchos_combined() {
     xcrun lipo -create "$watchsimulator_path/$binary_path" "$watchos_path/$binary_path" -output "$out_path/$product_name/$module_name"
 }
 
+xcrealm_work_around_rdar_23055637() {
+    # xcodebuild times out waiting for the iOS simulator to launch if it takes > 120 seconds for the tests to
+    # build (<http://openradar.appspot.com/23055637>). Work around this by having the test phases intentionally
+    # exit after they finish building the first time, then run the tests for real.
+    REALM_EXIT_AFTER_BUILDING_TESTS=YES xcrealm "$1" || true
```

Comment:
It's used in Realm.xcodeproj/project.pbxproj. GitHub is hiding it from the diff since so many lines have changed.